data/coalition/coalition missions.txt

```diff
@@ -146,6 +146,77 @@ mission "Coalition License Patch"
+event "lost in coalition"
+
+
+
+mission "Coalition: Lost: Marker"
+	name "Foreign Aid"
+	description "The Heliarchs have noticed that you've lost your jump drive, being now stranded in Coalition space, and have offered to help. Head to <destination> to speak with them."
+	landing
+	source
+		near "Ekuarik" 1 100
+	destination "Ring of Wisdom"
+	to offer
+		has "event: lost in coalition"
+	on offer
+		conversation
+			`Another landing, another day spent stranded in this alien region, with no other humans in sight. Be it with the help of a Heliarch agent, via their own translation devices, or just by gesturing and hoping for the best, you've managed to get by just fine, but the thought that you may live the rest of your life living among an alien civilization refuses to leave your mind.`
+			`	Trying to take your mind off of it all, you start walking around <origin>, looking for some place to eat, but are stopped by a Heliarch delegation shortly after you leave your ship.`
+			`	"Cautiously observed during the past few weeks, scans of your ship have been, Captain <last>," the Saryd begins.`
+			`	"Unlike previous scans, a jump drive in your ship, our scanners found not," the Kimek continues.`
+			`	"Though appreciate the work you here perform, we do, for you to be cut-off from the galaxy, unlike the Quarg we wish not," the Arach says. "Discussed much, our elders have, and decided to help you, they have. To <destination> you must go, there discussed, the terms will be."`
+			`	You thank them for bringing you this information, and go back to your ship to prepare a route to <destination>.`
+	to complete
+		has "Coalition: Lost: Offer: active"
+
+
+
+mission "Coalition: Lost: Offer"
+	name "Jump Drive Loan"
+	description "The Heliarchs have provided you a jump drive to leave Coalition space. Bring another jump drive to <destination> by <date>, to replace the one you borrowed, or they will be extremely angry by you breaking their trust."
+	deadline 333
+	landing
+	source "Ring of Wisdom"
+	to offer
+		has "Coalition: Lost: Marker: active"
+	on offer
+		conversation
+			`The Heliarchs are waiting for you when you land. You see no civilians as you look around, but there are enough armed guards to man several of their Punishers. Said Punishers are all parked near your own ship.`
+			`	The Saryd starts speaking, "As to be trusted, you have been deemed, provide you with a jump drive, so that stranded you are not, we will."`
+			`	"Mind you, not a gift this is, but a loan, Captain," the Kimek continues. "Another one to replace this one, you must bring us, so that losing a copy of such valuable equipment, we are not."`
+			`	"By <date>, brought here the replacement jump drive must be. If any longer you take, assume that betrayed our trust you have, we will, and an enemy, considered you will be," the Arach finishes explaining. "Agree with those terms, do you, Captain?"`
+			choice
+				`	"Yes, I promise I'll be back here with a second jump drive by <date>."`
```

Comment:
It seems odd to me for the player to repeat the date, especially given that it is reiterated in the next line, so I'd remove that part.
```suggestion
				`	"Yes, I promise I'll return with a second jump drive."`
```